src/test/java/core/basesyntax/FruitShopServiceTest.java

```diff
@@ -0,0 +1,118 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.FruitsStorage;
+import core.basesyntax.exception.ValidationException;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.OperationType;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.service.impl.FruitShopServiceImpl;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.impl.BalanceOperationHandlerImpl;
+import core.basesyntax.strategy.impl.OperationStrategyImpl;
+import core.basesyntax.strategy.impl.PurchaseOperationHandlerImpl;
+import core.basesyntax.strategy.impl.ReturnOperationHandlerImpl;
+import core.basesyntax.strategy.impl.SupplyOperationHandlerImpl;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitShopServiceTest {
+    private static OperationStrategy operationStrategy;
+    private static List<FruitTransaction> fruitTransactionList;
+    private static Map<String, Integer> fruitsStorageExpected;
+    private static Map<OperationType, OperationHandler> operationTypeMap;
+
+    @BeforeClass
+    public static void beforeClass() {
+        FruitTransaction fruitTransaction1 = new FruitTransaction(OperationType.BALANCE,
```

Comment:
It is better use more informative names
Using numbers in names is bad practice, fix everywhere
```suggestion
        FruitTransaction bananaForBalance = new FruitTransaction(OperationType.BALANCE,
```

Reply:
Ok, thanks!